tests/runtests.py

```diff
@@ -44,31 +53,38 @@ def upath(path):
     'django.contrib.messages',
     'django.contrib.admin',
     'django.contrib.staticfiles',
-    "mezzanine.boot",
-    "mezzanine.conf",
-    "mezzanine.core",
-    "mezzanine.generic",
-    "mezzanine.pages",
-    "mezzanine.forms",
-    'django.contrib.comments',
-    'filebrowser_safe',
-    'grappelli_safe',
+]
+
+if is_mezzanine_installed():
```

Comment:
I noted [here](https://github.com/fusionbox/django-widgy/pull/294#issuecomment-69395162) that I was not entirely happy with some of the hoops that I had to implement in order to run the tests across different dependencies.  Very open to an alternate suggestion for how I should accomplish this.  I chose this approach because the end result was less code refactoring which I know is sometime unwelcome.